.github/README.md

```diff
@@ -67,8 +67,6 @@ The Oppia Android code is released under the [Apache v2 license](https://github.
   * [Discussion forum](http://groups.google.com/group/oppia)
```

Comment:
Could you please change this to the android github discussions page?

Reply:
Yes sure.

Reply:
Done

Reply:
This looks to be resolved now.